### DISCUSSION TITLE
v2.6.7 Remote-Build deserialize failed

### DISCUSSION THREAD

**Discussion by HowardWang12138** [xmake-io/xmake#2408] [Q&A]
+ client: linux centos7   
+ server: linux redhat     

deserialize allmost all files failed in server. 

## eg:
error: @programdir/modules/private/service/server.lua:303 cannot deserialize "add_rule(\"mode.debu....", <eof> expected near 'set_languages'  
traceback:
@programdir/modules/private/service/server.lua:303: in metond recv_object
@.....

此外报错还包括一些头文件中的防卫式声明， 以及 c/c++ src， xmake.lua ,   .sh,  .cmake  等多种文件， 经检查

**Comment by waruqi** [comment]
提供下两端的 -vD logs
